strings-csharp/StringFormattableString/StringFormattableStringTests/FormattableStringsUnitTests.cs

```diff
@@ -0,0 +1,53 @@
+using StringFormattableString;
+using System.Runtime.CompilerServices;
+
+namespace StringFormattableStringTests
+{
+    [TestClass]
+    public class FormattableStringsUnitTests
+    {
+        private readonly FormattableStringsMethods _formattableStringMethods;
+
+        public FormattableStringsUnitTests()
+        {
+            _formattableStringMethods = new FormattableStringsMethods();
+        }
```

Comment:
This could be simplified into a member initializer:
`private readonly FormattableStringsMethods _formattableStringMethods = new();`